puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/pipeline/forklift-centos-devel-pipeline.yaml

```diff
@@ -0,0 +1,15 @@
+
+- job:
+    name: forklift-centos-devel-pipeline
+    project-type: pipeline
+    triggers:
+      - reverse:
+          jobs:
+            - foreman-forklift-centos-devel-test
+          result: success
```

Comment:
This trigger won't do what you think it does and you really only need the timed one.